frontend/src/components/LandingPage/index.vue

```diff
@@ -60,7 +60,7 @@ const handleOk = () => {
                 >Try For Free</a-button
               >
               <LandingPageBtn
-                href="https://www.openmldb.com"
+                href="https://www.openmldb.ai"
```

Comment:
![image](https://github.com/4paradigm/FeatInsight/assets/11715200/3e62d78c-835d-4746-802c-8b99c5148bae)

do not add the `www` prefix